modules/monitoring-optimizing-alerting-rules-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="Optimizing-alerting-rules-for-core-platform-monitoring{context}"]
+= Optimizing alerting rules for core platform monitoring
```

Comment:
Do we need to start a concept heading with a noun or noun phrase rather than a gerund?
I'm looking at https://redhat-documentation.github.io/modular-docs/#creating-concept-modules and https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-concepts. How about this?
```suggestion
= Optimization of alerting rules for core platform monitoring
```

Reply:
Another good catch. You are correct. I was trying to stay consistent with the parallel topic for user workload monitoring: https://docs.openshift.com/container-platform/4.13/monitoring/managing-alerts.html#Optimizing-alerting-for-user-defined-projects_managing-alerts. But I think the new one needs to follow the guidelines even though the old one does not. I think I'll change this one to "Tips for optimizing alerting rules for core platform monitoring". Because I think this is less awkward than "Optimization of..."

Reply:
Changed.